demosplan/DemosPlanCoreBundle/Controller/Procedure/DemosPlanProcedureController.php

```diff
@@ -2781,11 +2809,25 @@ protected function handleDeleteBoilerplate(string $boilerplateId)
      *
      * @throws \demosplan\DemosPlanCoreBundle\Exception\MessageBagException
      */
-    protected function handleDeleteBoilerplateGroup(string $boilerplateGroupId)
-    {
+    protected function handleDeleteBoilerplateGroup(
+        string $boilerplateGroupId,
+        BoilerplateRepository $boilerplateRepository
+    ) {
+        $boilerplatesOfGroupToDelete = new ArrayCollection();
         $boilerplateGroupToDelete = $this->procedureService->getBoilerplateGroup($boilerplateGroupId);
+        if (null !== $boilerplateGroupToDelete) {
+            foreach ($boilerplateGroupToDelete->getBoilerplates() as $boilerplate) {
+                $boilerplatesOfGroupToDelete->add($boilerplate);
+            }
+        }
+
         $title = null === $boilerplateGroupToDelete ? '' : $boilerplateGroupToDelete->getTitle();
```

Comment:
I think that $successfully is false when $boilerplateGroupToDelete is null. For this reason I think we can remove the $title here and can $boilerplateGroupToDelete->getTitle() move to the MessageBag directly.

Reply:
I'm wrong we delete $boilerplateGroupToDelete before we add the message to the MessageBag so it is null.